### PR TITLE
minor spelling tweaks

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

**Comment by brettkoonce** [comment]
See also #107.

**Comment by bddppq** [comment]
@brettkoonce The `Operators.md` file is generated, could you run `python onnx/defs/gen_doc.py` to regenerate it?

**Comment by brettkoonce** [comment]
Can't figure out where these are coming from:

usethe => use the
GlobalMaxPooll => extra l?
flatenned => flattened

File: docs/Operators.md [hunk 1]
```diff
@@ -127,7 +127,7 @@
   * **input**:
     <dl>
       <dt>X</dt>
-      <dd>Input data tensor from the previous operator; dimensions for image case are (N x C x H x W), where N is the batch size, C is the number of channels, and H and W are the height and the width of the data. For non image case, the dimension are in the form of (N x C x D1 x D2 ... Dn), where N is the batch size.</dd>
```
Comment (bddppq) [inline docs/Operators.md]: This doesn't match the current `AveragePool` doc at https://github.com/onnx/onnx/blob/master/onnx/defs/nn/defs.cc#L35. Is it possible your working copy is outdated? Note after changing C++ files you need to reinstall in order to get the changes effective.

Comment (ezyang) [reply docs/Operators.md]: This sounds like a good reason to move the spec files out of C++ :)

Comment (bddppq) [reply docs/Operators.md]: On the other hand you want to have these operators schema in c++ for being able to do validation on any environment. :-)

Comment (brettkoonce) [reply docs/Operators.md]: which files do I need to re-install/what is being pulled in exactly here?

Comment (bddppq) [reply docs/Operators.md]: How did you install onnx at the beginning? The procedure of changing docs in `Operators.md` is first update the `*.defs.cc` file, and then install the new sources into your python environment (there is a "Source" section in our repo frontpage telling you how to install from source) and then run `python onnx/defs/gen_doc.py` to get the new `Operators.md` file.

**Comment by brettkoonce** [comment]
@bddppq thanks for the tips!

**Comment by bddppq** [comment]
Looks good! Could you also check in the `*defs.cc` source files changes?

**Comment by brettkoonce** [comment]
@bddppq I tried, I don't seen to be able to find any corresponding code?

**Comment by bddppq** [comment]
I merged master to your PR and then the changes in `Operators.md`disappeared.

**Comment by bddppq** [comment]
Thanks for your contribution!

**Comment by brettkoonce** [comment]
@bddppq Ahh, that makes sense.

**Pull request merged**